R/recipes.R

```diff
@@ -84,6 +84,8 @@ model_lin_recipe <- function(data, pred_vars, cat_vars,
     # Remove any variables not an outcome var or in the pred_vars vector
     step_rm(any_of("time_split")) %>%
     step_rm(-all_outcomes(), -all_predictors(), -has_role("ID")) %>%
+    # Drop extra location predictors that aren't nbhd or township
+    step_rm(starts_with("loc_"), -all_numeric_predictors()) %>%
```

Comment:
This is just a baseline model, not the real one, so just like in the res model we drop extraneous categorical location features.